src/app/public/modules/autocomplete/autocomplete.component.ts

```diff
@@ -213,9 +222,16 @@ export class SkyAutocompleteComponent
   @Input()
   public showAddButton: boolean = false;
 
+  /**
+   * @internal
+   * Shows the Show more button in the actions bar
+   */
+  @Input()
+  public showMoreButton: boolean = false;
```

Comment:
I'll touch base with you about this one.... this is a case where I wish I had named the `showAddButton` something different in the previous PR but it has already been released. You are correct in that I could rename this in `autocomplete` but I can't rename the `showAddButton` that exists on lookup itself because it is not internal in that case. I'll ping you about this.